Dockerfile

```diff
@@ -3,6 +3,20 @@ FROM python:3.6
 # Development build
 ARG IS_NOT_PRODUCTION
 
+# Build-time metadata as defined at http://label-schema.org
+ARG BUILD_DATE
+ARG VCS_REF
+ARG VERSION
+LABEL org.label-schema.build-date=$BUILD_DATE \
```

Comment:
should we use quotes around variables?

Reply:
Not sure. I've checked Docker's documentation.